netbox_slm/models.py

```diff
@@ -4,6 +4,7 @@
 
 from netbox.models import NetBoxModel
 from utilities.querysets import RestrictedQuerySet
+from utilities.forms.fields import CommentField
```

Comment:
This seems to be unused, was probably intended elsewhere?

Reply:
this was a mistake and shouldnt be pushed.
I will remove it with the next commit